eng/Versions.props

```diff
@@ -282,6 +282,11 @@
     <MicrosoftSourceLinkGitLabVersion>9.0.0-beta.25072.1</MicrosoftSourceLinkGitLabVersion>
     <MicrosoftSourceLinkBitbucketGitVersion>9.0.0-beta.25072.1</MicrosoftSourceLinkBitbucketGitVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <SystemBuffersVersion>4.6.0</SystemBuffersVersion>
+    <SystemMemoryVersion>4.6.0</SystemMemoryVersion>
+    <SystemThreadingTasksExtensionsVersion>4.6.0</SystemThreadingTasksExtensionsVersion>
+  </PropertyGroup>
```

Comment:
I don't think that you can already unconditionally use these versions. VS hasn't picked those versions up yet: https://github.com/dotnet/msbuild/pull/11038

Either downgrade to the previous versions (when not building from source) or make sure that desktop msbuild tasks use the older versions.

Reply:
@dsplaisted I'm relatively sure that the above error that you posted is due to this dependency update. This manifests in a `MissingMethodException` as types from those packages are used in the public method signature: `System.ReadOnlySpan`.